File: app/src/main/java/com/anenha/weather/app/model/TranslateResponseModel.java
package com.anenha.weather.app.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by ana.j.saragossa on 10/24/17.
 */

public class TranslateResponseModel {
    @SerializedName("extract")
    @Expose
    private Extract extract;
    @SerializedName("originalResponse")
    @Expose
    private String originalResponse;

    public Extract getExtract() {
        return extract;
    }

    public void setExtract(Extract extract) {
        this.extract = extract;
    }

    public String getOriginalResponse() {
        return originalResponse;
    }

    public void setOriginalResponse(String originalResponse) {
        this.originalResponse = originalResponse;
    }
}

File: app/src/main/java/com/anenha/weather/app/model/TranslateResponseModel.kt
package com.anenha.weather.app.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by ana.j.saragossa on 10/24/17.
 */

class TranslateResponseModel {
    @SerializedName("extract")
    @Expose
    var extract: ExtractModel? = null
    @SerializedName("originalResponse")
    @Expose
    var originalResponse: String = ""
}
